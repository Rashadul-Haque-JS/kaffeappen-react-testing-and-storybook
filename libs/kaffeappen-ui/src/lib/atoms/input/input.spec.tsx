import { render } from '@testing-library/react';
import InputComponent from './input';


describe('Input field', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<InputComponent placeholder={'användarnamn'} type={'text'}/>);
    expect(baseElement).toBeTruthy();
  });

  it('should match snapshot successfully', () => {
    const { baseElement } = render(<InputComponent  placeholder={'lösenord'} type={'password'}/>);
    expect(baseElement).toMatchSnapshot();
  });
});