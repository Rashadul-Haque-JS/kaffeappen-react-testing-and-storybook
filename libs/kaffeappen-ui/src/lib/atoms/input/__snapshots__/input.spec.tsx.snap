// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input field should match snapshot successfully 1`] = `
<body>
  <div>
    <input
      class="input__StyledInput-sc-1rotb0l-0 HWiIs"
      placeholder="lÃ¶senord"
      required=""
      type="password"
    />
  </div>
</body>
`;
